src/Framework/src/Microsoft.AspNetCore.App.Runtime.csproj

```diff
@@ -356,6 +356,11 @@ This package is an internal implementation of the .NET Core SDK and is not meant
           EnvironmentVariables="COMPlus_PartialNGen=0"
           IgnoreStandardErrorWarningFormat="true"
           StandardOutputImportance="High" />
+
+    <!-- crossgen tool generates e.g. *.ni.{53ae4659-8d91-47f1-8eb3-a458d3aa69ee}.map cross-plat. -->
```

Comment:
@tmat the problem I'm trying to workaround here is lack of information in the Windows PDBs VS downloads for `crossgen`ed assemblies in the Microsoft.AspNetCore.App.Runtime&hellip; packages. The symbol cache contains only the *.pdb file from before `crossgen`ing the assemblies though those packages contain both *.pdb (managed) and *.ni.pdb (`crossgen`ed) files. And, it seems like something gets confused when both files are available (perhaps the symbol server?). Does this fix make sense to you?

Separately, the Linux and macOS symbol packages contain both *.pdb (managed) and files with names like `*.ni.{53ae4659-8d91-47f1-8eb3-a458d3aa69ee}.map`. Are the symbol servers and VS Code or VS for Mac smart enough to use both?